regression/scripts/weights/eVTOL_Weights_Buildup_Regression.py

```diff
@@ -1,364 +0,0 @@
-# eVTOL_Weights_Buildup_Regression.py
-#
-# Created: Feb, 2018, J. Smart
-
-#------------------------------------------------------------------------------
-# Imports
-#------------------------------------------------------------------------------
-
-import sys
```

Comment:
wah, lets fix this ASAP

Reply:
It's coming back in when I pull in the VTOL branch